assets/js/admin/admin-scripts.js

```diff
@@ -9,6 +9,8 @@
  */
 
 jQuery.noConflict();
+// Provided access to golbal level
+var givesettingedit = false;
```

Comment:
Please make it more legible with underscores.

Reply:
Done